test/isochrone.cc

```diff
@@ -100,23 +100,23 @@ TEST(Isochrones, Basic) {
     const auto request =
         R"({"locations":[{"lat":52.078937,"lon":5.115321}],"costing":"auto","contours":[{"time":9}],"polygons":false,"denoise":0.2,"generalize":100})";
```

Comment:
I see we've updated the tests. Are these tests exercising the "long edge" issue? If yes, then :+1: , if no, then maybe add a test (gurka or otherwise)?

Reply:
Gurka testing it is a good idea. I'll do it 👍

Reply:
@mandeepsandhu gurka tests are added